soc/nxp/imxrt/rt11xx/Kconfig.soc

```diff
@@ -0,0 +1,99 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_11XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_SERIES
+	default "rt11xx" if SOC_SERIES_IMXRT_11XX
```

Comment:
as above

Reply:
it also matches

Reply:
No it does not `default "rt11xx" if SOC_SERIES_IMXRT_11XX`: `rt11xx` and `IMXRT_11XX` so either: `default "rt11xx" if SOC_SERIES_RT11XX` or `default "imxrt_11xx" if SOC_SERIES_IMXRT_11XX`

Reply:
You're talking about the name of the kconfig? Why

Reply:
Because these will be auto-generated by the build system in the future

Reply:
okay I changed the kconfig name treewide, the soc.yml, the string value, and the folder names